pkg/controller/postgresql.go

```diff
@@ -429,10 +420,21 @@ func (c *Controller) queueClusterEvent(informerOldSpec, informerNewSpec *acidv1.
 		clusterError = informerNewSpec.Error
 	}
 
-	if clusterError != "" && eventType != EventDelete && eventType != EventAdd {
-		c.logger.
-			WithField("cluster-name", clusterName).
-			Debugf("skipping %q event for the invalid cluster: %s", eventType, clusterError)
+	if clusterError != "" && eventType != EventDelete {
+		c.logger.WithField("cluster-name", clusterName).Debugf("skipping %q event for the invalid cluster: %s", eventType, clusterError)
+
+		switch eventType {
+		case EventAdd:
```

Comment:
is this Go format style no indentation for case?

Reply:
[yes](https://gobyexample.com/switch) :smiley: